### PR TITLE
implement bce loss

### PR DESCRIPTION
**Pull request by PososikTeam**

### CODE REVIEW

File: torchok/losses/classification/binary_cross_entropy.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+import torch.nn as nn
+import torch.nn.functional as F
+
+from typing import Union
+
+from torchok.constructor import LOSSES
+
+
+@LOSSES.register_class
+class BCEWithLogitsLossX(nn.BCEWithLogitsLoss):
```
Comment (Animatory) [inline torchok/losses/classification/binary_cross_entropy.py:13]: Rename to BCEWithLogitsLoss as it extends the original BCE without changing other functionality
